README.md

```diff
@@ -460,6 +494,27 @@ cygnusagent.sinks.mysql-sink.mysql_password = xxxxxxxxxxxx
 # how the attributes are stored, either per row either per column (row, column)
 cygnusagent.sinks.mysql-sink.attr_persistence = column
 
+# ============================================
+# OrionMongoSink configuration
+# sink class, must not be changed
+cygnusagent.sinks.mongo-sink.type = com.telefonica.iot.cygnus.sinks.OrionMongoSink
+# channel name from where to read notification events
+cygnusagent.sinks.mongo-sink.channel = mongo-channel
+# the FQDN/IP address where the MongoDB server runs
+cygnusagent.sinks.mongo-sink.mongo_host = x.y.z.w
+# the port where the MongoDB server listens for incomming connections
+cygnusagent.sinks.mongo-sink.mongo_port = 27017
+# a valid user in the MongoDB server
+cygnusagent.sinks.mongo-sink.mongo_username = mongo_username
+# password for the user above
+cygnusagent.sinks.mongo-sink.mongo_password = xxxxxxxx
+# data model (collection-per-service-path, collection-per-entity, collection-per-attribute)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/6133c3124462f6bac841b3fc6e4fc099baab4cf6